src/database/queries/baseFields/createOrUpdateByShortcode.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO base_fields (
+  label,
+  description,
+  short_code,
+  data_type,
+  scope
+)
+VALUES (
+  :label,
+  :description,
+  :shortCode,
+  :dataType,
+  :scope
+)
+ON CONFLICT (short_code)
+DO UPDATE SET
+  label = EXCLUDED.label,
+  description = EXCLUDED.description,
+  data_type = EXCLUDED.data_type,
+  scope = EXCLUDED.scope
+WHERE EXCLUDED.short_code = :shortCode
```

Comment:
To expand on that a little bit, both we and the database already know that `EXCLUDED.short_code = :shortCode` because of the `ON CONFLICT (short_code)`; if they differed, it wouldn't be a conflict, and there would be nothing to update.

Including this `WHERE` clause is redundant, and leads to confusion about how this works - like "wait, is that something we need to be careful of?"